Tribler/Core/Utilities/utilities.py

```diff
@@ -185,6 +186,10 @@ def isValidURL(url):
     return True
 
 
+def requests_get(url, params=None, **kwargs):
+    return threads.deferToThread(requests.get, url, params=params, **kwargs)
```

Comment:
That should be using twisted's async request system instead of pushing a blocking version of it to the threadpool